plugins/woocommerce-blocks/assets/js/atomic/utils/blocks-registration-manager/template-change-detector.ts

```diff
@@ -3,7 +3,12 @@
  */
 import { subscribe } from '@wordpress/data';
 import { getPath, getQueryArg } from '@wordpress/url';
-import { isNumber } from '@woocommerce/types';
+
+enum ContentType {
+	POST_OR_PAGE = 'post-or-page',
+	TEMPLATE = 'template',
+	NONE = 'none',
```

Comment:
Do you think for the sake of consistency and understanding to describe these we should use post type values?